### PR TITLE
bug 2083219: Support c1.. in interface name

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

**Comment by bn222** [comment]
@pliurh

**Comment by pliurh** [comment]
Can we derive the PF-rep name from the PF we select by `sriov_conf.json`? Then we don't need to render `.PfRepName` anymore.

File: bindata/machine-config/ovs-units/ovs-configuration.service.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ dropins:
   contents: |
     [Service]
     ExecStartPre=/usr/local/bin/ovs-reconfig.sh
-    ExecStartPost=/bin/ovs-vsctl --may-exist add-port br-ex {{.PfRepName}}
+    ExecStartPost=/usr/local/bin/ovs-add-pf.sh {{.PfRepName}}
```
Comment (pliurh) [inline bindata/machine-config/ovs-units/ovs-configuration.service.yaml:7]: I think we can just remove `.PfRepName`? Or do you want to keep it just for development usage?

Comment (bn222) [reply bindata/machine-config/ovs-units/ovs-configuration.service.yaml:7]: Currently, the way I've made this work is that `.PfRepName` can be used to override the selection. Let's keep it for development purposes. We can always (easily) remove it later.

**Pull request merged**